### PR TITLE
feat(tutorial): complete step 5

### PR DESCRIPTION
**Pull request by pfine**
Closes #

URL: https://carbon-tutorial-pfine.mybluemix.net/#/

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

**Comment by pfine** [comment]
URL added to description:
https://carbon-tutorial-pfine.mybluemix.net/#/

File: src/index.scss [hunk 1]
```diff
@@ -1,9 +1,19 @@
-$feature-flags: (
```
Comment (kellychurchill) [inline src/index.scss]: The `feature-flags` mapping should not be removed. Your layout is not rendering properly without it: https://carbon-tutorial-pfine.mybluemix.net/#/

Comment (pfine) [reply src/index.scss]: thanks, fixed and re-deployed :D

**Comment by pfine** [comment]
fixed, thanks

**Comment by kellychurchill** [comment]
👍  Looks good! 
Feel free to stop this running application on IBM Cloud to avoid the possibility of incurring any charges 💵

**Pull request closed without merging**